### PR TITLE
Update v1.0 docs for certificate based auth configuration

### PR DESCRIPTION
**Pull request by eketo-msft**

### CODE REVIEW

File: api-reference/v1.0/api/certificatebasedauthconfiguration-delete.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+```http
+DELETE /organization/{id}/certificateBasedAuthConfiguration/{id}
+```
+
+## Request headers
+
+| Name          | Description   |
+|:--------------|:--------------|
+| Authorization | Bearer {token} |
+| Content-Type | application/json |
```
Comment (Lauragra) [inline api-reference/v1.0/api/certificatebasedauthconfiguration-delete.md]: Why are we listing this header if the request doesn't include a JSON request body?

File: api-reference/v1.0/api/certificatebasedauthconfiguration-delete.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+```http
+DELETE /organization/{id}/certificateBasedAuthConfiguration/{id}
+```
+
+## Request headers
+
+| Name          | Description   |
+|:--------------|:--------------|
+| Authorization | Bearer {token} |
```
Comment (Lauragra) [inline api-reference/v1.0/api/certificatebasedauthconfiguration-delete.md]: Global: Please indicate that the Authorization header is required. The description should be "Bearer {token}. Required."

File: api-reference/v1.0/api/certificatebasedauthconfiguration-delete.md (hunk 1 above)
Comment (eketo-msft) [reply api-reference/v1.0/api/certificatebasedauthconfiguration-delete.md]: I just had another reviewer ask why we don't doc it for gets, sigh.  At least we are consistently inconsistent in feedback.

File: api-reference/v1.0/api/certificatebasedauthconfiguration-list.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+```http
+GET /organization/{id}/certificateBasedAuthConfiguration
+```
+
+## Request headers
+
+| Name      |Description|
+|:----------|:----------|
+| Authorization | Bearer {token} |
+| Content-Type | application/json |
```
Comment (Lauragra) [inline api-reference/v1.0/api/certificatebasedauthconfiguration-list.md]: Same comment here - remove the content-type header, and add required to Authorization.

File: api-reference/v1.0/api/certificatebasedauthconfiguration-delete.md (hunk 2 above)
Comment (eketo-msft) [reply api-reference/v1.0/api/certificatebasedauthconfiguration-delete.md]: Is this new?

Comment (eketo-msft) [reply api-reference/v1.0/api/certificatebasedauthconfiguration-delete.md]: Oh wow, our docs are all over the place on this.

File: api-reference/v1.0/api/certificatebasedauthconfiguration-get.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+```http
+GET /organization/{id}/certificateBasedAuthConfiguration/{id}
+```
+
+## Request headers
+
+| Name      |Description|
+|:----------|:----------|
+| Authorization | Bearer {token} |
+| Content-Type | application/json |
```
Comment (Lauragra) [inline api-reference/v1.0/api/certificatebasedauthconfiguration-get.md]: Looks like we need to remove Content-type here as well.

**Pull request merged**